hoot-core/src/main/cpp/hoot/core/elements/ElementDeduplicator.cpp

```diff
@@ -64,9 +64,9 @@ void ElementDeduplicator::dedupe(OsmMapPtr map)
   LOG_INFO("De-duping intra-map: " << map->getName() << "...");
   LOG_DEBUG(map->getName() << " size before de-duping: " << map->size());
 
-  const long nodesBefore = map->getNodeCount();
-  const long waysBefore = map->getWayCount();
-  const long relationsBefore = map->getRelationCount();
+  const int nodesBefore = map->getNodeCount();
+  const int waysBefore = map->getWayCount();
+  const int relationsBefore = map->getRelationCount();
```

Comment:
All of these functions return a `long`, the more precise way to do this is further down when `nodesBefore` is used.  Since it is calculating the difference between two longs, this would be the place to do it.
```
_map1DuplicateNodesRemoved = (int)(nodesBefore - map->getNodeCount());
//  or
_map1DuplicateNodesRemoved = std::static_cast<int>(nodesBefore - map->getNodeCount());
```